packages/@lwc/engine-server/src/__tests__/fixtures/api/setter/modules/x/child/child.js

```diff
@@ -0,0 +1,11 @@
+import { LightningElement, api } from 'lwc';
+
+export default class Child extends LightningElement {
+    @api set setterGetterApi(value) {
```

Comment:
```suggestion
    @api set setter(value) {
```

Can we also add a `@api get getter` in this test, to make sure we cover both?